### PR TITLE
Navigate to the correct query hyperlink.

### PR DESCRIPTION
**Pull request by abhishekrb19**
Minor doc update that will take the user to the appropriate query hyperlink.

### CODE REVIEW

**Comment by vogievetsky** [comment]
good one 👍

**Comment by abhishekrb19** [comment]
Hi @vogievetsky , I am not sure why the Inspections check failed. Can't view details as I don't have a TeamCity account

File: docs/querying/querying.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Druid has numerous query types for various use cases. Queries are composed of va
 
 ## Which query should I use?
 
-Where possible, we recommend using [Timeseries]() and [TopN]() queries instead of [GroupBy](). GroupBy is the most flexible Druid query, but also has the poorest performance.
+Where possible, we recommend using [Timeseries](../querying/timeseriesquery.md) and [TopN](../querying/topnquery.md) queries instead of [GroupBy](../querying/groupbyquery.md). GroupBy is the most flexible Druid query, but also has the poorest performance.
```
Comment (fjy) [inline docs/querying/querying.md]: I think this description is out of date and this whole section should be deleted.

Comment (abhishekrb19) [reply docs/querying/querying.md]: @fjy, are you saying that this recommendation is _not_ true anymore?

**Pull request closed without merging**